support-frontend/assets/helpers/productPrice/productOptions.ts

```diff
@@ -38,22 +36,20 @@ export type PaperProductOptions =
 	| typeof Sixday
 	| typeof Everyday
 	| typeof SaturdayPlus
-	| typeof SundayPlus
 	| typeof WeekendPlus
 	| typeof SixdayPlus
 	| typeof EverydayPlus;
 
 const ActivePaperProductTypes: readonly PaperProductOptions[] = [
+	EverydayPlus,
```

Comment:
I've moved the plus plans to the top of this list because the ordering controls the order they appear in on the newspaper landing page